vnext/PropertySheets/External/Microsoft.ReactNative.Uwp.CSharpApp.SourceReferences.targets

```diff
@@ -14,7 +14,5 @@
       <Project>{F2824844-CE15-4242-9420-308923CD76C3}</Project>
```

Comment:
Does this need to be a separate file for each template type?
Can't a shared file be deployed for both C# App, C# Lib, etc?

Reply:
The references have to be private with libs, but not with the app. I didn't think we had anything to key off of to have it in one file, so I just copied the existing one.